### PR TITLE
Enable sandboxing for Python manual and upgrade reviews

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Enable sandboxing for Python and upgrade python api review parser version

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
@tjprescott I have included Python parser upgrade as part of change to enable sandboxing so reviews will be upgraded using ubuntu platform.

**Comment by praveenkuttappan** [comment]
Fixes #4657 https://github.com/Azure/azure-sdk-tools/issues/4655

File: tatus [hunk 1]
```diff
@@ -0,0 +1,64 @@
+[1mdiff --git a/eng/pipelines/apiview-review-gen-python.yml b/eng/pipelines/apiview-review-gen-python.yml[m
```
Comment (weshaggard) [inline tatus:1]: I assume you didn't include this file intentionally.

Comment (praveenkuttappan) [reply tatus:1]: Thanks for catching it. This was accidental add. :(

**Review by weshaggard — APPROVED** [review]

**Review by tjprescott — APPROVED** [review]

**Pull request merged**